tests/src/main/java/net/neoforged/neoforge/debug/client/ClientEventTests.java

```diff
@@ -31,4 +38,20 @@ static void playerClientChatEvent(final ClientChatEvent event, final DynamicTest
     static void clientPlayerChangeGameTypeEvent(final ClientPlayerChangeGameTypeEvent event, final DynamicTest test) {
         test.requestConfirmation(Minecraft.getInstance().player, Component.literal("Did you just change your game mode from " + event.getCurrentGameType() + " to " + event.getNewGameType() + "?"));
     }
+
+    @TestHolder(description = { "Tests if the RegisterRenderBuffersEvent event is fired and whether the registered render buffer is represented within a fixed render buffer map" }, enabledByDefault = true)
+    static void registerRenderBuffersEvent(final DynamicTest test) {
+        test.framework().modEventBus().addListener((final RegisterRenderBuffersEvent event) -> {
+            event.registerRenderBuffer(RenderType.lightning());
+        });
+        test.framework().modEventBus().addListener((final RenderLevelStageEvent.RegisterStageEvent event) -> {
+            Map<RenderType, BufferBuilder> fixedBuffers = ObfuscationReflectionHelper.getPrivateValue(MultiBufferSource.BufferSource.class, Minecraft.getInstance().renderBuffers().bufferSource(), "fixedBuffers");
```

Comment:
Would it be possible to write a test that doesn't use the reflection helper? (That we have plans to phase out).